68. Text Justification/psycher.py

```diff
@@ -0,0 +1,35 @@
+# Accepted
+# submitted at Oct 29, 2024 18:04
+# Runtime: 0ms     Beats 100.00%
+# Memory:  16.60MB Beats 63.74%
+
+# This is a refined solution
+# I submitted one that wasn't as clean and browsed other solutions befor rendering this final version
+# I got the idea of comparing remains with the word index from one of the solutions (line 32)
+
+class Solution:
+   def fullJustify(self, words: List[str], maxWidth: int) -> List[str]:
+           out = []
+           while len(words) > 0:
+               count = maxWidth
+               curr_line = []
+               while len(words) > 0 and count >= len(words[0]):
+                   w = words.pop(0)
```

Comment:
Interesting that you could still get a good runtime with this `O(n^2)` operation here.